python/sqlflow_playground/playground_server_design.md

```diff
@@ -0,0 +1,136 @@
+# SQLFlow Playground Server
+
+SQLFlow Playground Server exposes a REST API service that enables users to share
+the resources in one playground cluster. Users can take advantage of SQLFlow by
+installing a small plugin on her/his Jupyter Notebook.
```

Comment:
Which plugin, maybe we can link to it. And do we also need to do some work on this plugin?

Reply:
It's our pysqlflow plugin, that will be in another PR on that repo.